Packages/com.unity.inputsystem/InputSystem/InputSystem.cs

```diff
@@ -3632,15 +3675,18 @@ internal static void TrackDirtyInputActionAsset(InputActionAsset asset)
         }
 
 #else
-        private static void InitializeInPlayer(IInputRuntime runtime = null, InputSettings settings = null)
+        private static void InitializeInPlayer(IInputRuntime runtime = null, InputSettings settings = null, InputActionAsset actions = null)
         {
             if (settings == null)
                 settings = Resources.FindObjectsOfTypeAll<InputSettings>().FirstOrDefault() ?? ScriptableObject.CreateInstance<InputSettings>();
 
+            if (actions == null)
+                actions = Resources.FindObjectsOfTypeAll<InputActionAsset>().FirstOrDefault();
```

Comment:
Do we really want to pick one if a user has set this to null.
I don't think we do for the actions.

Reply:
This was causing 4 tests to fail so I've removed it.

Reply:
Reverted - see later comment